pinot-core/src/main/java/com/linkedin/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -144,6 +144,12 @@ public void stop() {
     String tableNameWithType = queryRequest.getTableNameWithType();
     long numDocsScanned =
         Long.parseLong(dataTableMetadata.getOrDefault(DataTable.NUM_DOCS_SCANNED_METADATA_KEY, INVALID_NUM_SCANNED));
+    long numIndicesLoaded =
+        Long.parseLong(dataTableMetadata.getOrDefault(DataTable.NUM_INDICES_LOADED_KEY, "0"));
```

Comment:
Could we investigate if these can be stored as numeric values within the metadata, so parsing is not required?